src/Monolog/Handler/MongoDBHandler.php

```diff
@@ -11,55 +11,51 @@
 
 namespace Monolog\Handler;
 
-use Monolog\Logger;
-use Monolog\Formatter\NormalizerFormatter;
+use MongoDB\Driver\BulkWrite;
```

Comment:
FYI: This is why the original BulkWrite construction was failing (PHP was looking for a Monolog\Handler\BulkWrite class).